exercises/03-keywords_and_variables.md

```diff
@@ -144,8 +168,13 @@ Now that you've successfully changed to use `${URL}` variable, think about what
 change except the domain of the SUT. Create those variables and modify the test case to use them instead
 of hard coded values.
 
+- Consider what other values could be implemented as variables, add them to your variables table and
```

Comment:
Previously, we have also abstracted away `demo` and `mode` to `${USERNAME}` and `${PASSWORD}` (respectively) -- should we note that here? It will make it easier in the following exercises, where these credentials need to be in multiple places.

Reply:
Added an indented bullet to address this.